assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -659,6 +661,53 @@ export const DEFAULT_NOTIFICATIONS = {
 		isDismissible: true,
 		featureFlag: 'googleTagGateway',
 	},
+	[ GTG_AUTO_ENABLE_NOTIFICATION ]: {
+		Component: GoogleTagGatewayAutoEnableNotification,
+		priority: PRIORITY.INFO,
+		areaSlug: NOTIFICATION_AREAS.DASHBOARD_TOP,
+		groupID: NOTIFICATION_GROUPS.DEFAULT,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: async ( { select, resolveSelect, dispatch } ) => {
+			const isGTGModuleConnected =
+				select( CORE_SITE ).isAnyGoogleTagGatewayModuleConnected();
```

Comment:
I flagged this on another instance but this isn't entirely safe because module connected state may not be resolved (although it should be). This function runs once to determine if the notification is enqueued or not so we need to await values that may not be available yet in all cases within `checkRequirements`.